index.html

```diff
@@ -219,6 +219,25 @@ <h2 class="about-title">About <br> Myself</h2>
             <hr class="solid" />
           </div>
     </section>
+    <footer class="footer">
+      <div class="contact" id="Contact">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I'd love to help with it </p>
+      <form action="https://formspree.io/f/xwkydypy" id="form" class="contact-form" method="post" novalidate>
```

Comment:
- [ ] The `novalidate` attribute in the form tag is making the form to be able to be submitted without the required elements, so the `required` attributes don't work as expected. Please remove this attribute so the user cannot submit the form without using all the inputs.